apps/docs/content/tokens/getting-started/introduction.mdx

```diff
@@ -8,10 +8,41 @@ order: 1
 
 At the heart of Workleap Design language are tokens, which drive its functionality. They are the key ingredients to everything UI. Workleap brand is dissected into design tokens, they live in the Hopper Design System.
 
+<Card>
+    <Image src="https://assets.workleap.com/hopper/webdoc/token-details.png" width="654" height="45" alt="Token tiers diagram" />
+</Card>
+
 ## Definitions
 
 **Core tokens** are referring to raw values such as an hexadecimal color code, a border-width, a font-size. Think of them as paint name, Chocolate Frost is easier to use in a conversation than the exact color recipe.
 
 **Semantic tokens** adds the intention layer to tokens, they map to a core token, or tokens should we say as the semantic tier has a notion of themability. Using our Chocolate Frost paint color example this would dictate where the paint color should be used, on dry walls or plastic, inside or outside.
 
 **Component tokens** refers to the nuts and bolts of the components and their implementation details.  They are tied to their component. In our paint color example, this would be where a specific color is actually used, in which room and on which architectural part of it(ceiling, doors, wall).
+
+```json title="package.json"
```

Comment:
The title style seems broken here

<img width="797" alt="Capture d’écran, le 2023-07-26 à 20 46 29" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/7d858503-63e0-4d4b-853c-2b4d24e25814">